tests/unit/models/project-test.js

```diff
@@ -1,12 +1,69 @@
 import { moduleForModel, test } from 'ember-qunit';
+import Ember from 'ember';
 
 moduleForModel('project', 'Unit | Model | project', {
   // Specify the other units that are required for this test.
-  needs: ['model:project-category', 'model:organization', 'model:task', 'model:project-skill']
+  needs: ['model:project-category', 'model:organization', 
+          'model:organization-membership', 'model:user', 
+          'model:task', 'model:project-skill']
 });
 
 test('it exists', function(assert) {
   let model = this.subject();
   // let store = this.store();
   assert.ok(!!model);
 });
+
+test('it should has attributes', function(assert) {
+  let model = this.subject();
+  let attributes = Object.keys(model.toJSON());
```

Comment:
Is this a common approach here? I've never seen it, but I'm also not usually writing model tests.